### DISCUSSION TITLE
build (12.x) Expected — Waiting for status to be reported

### DISCUSSION THREAD

**Discussion by moroshko** [community/community#26233] [Actions]
<p>I used to run my workflow both on Node 10.x and 12.x.</p>
<p>Since I deleted Node 12.x from the workflow file, the 12.x is still run for some reason and it’s stuck with:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/3/38ab598293ab0c530d8e82dbd748b2c00a89a0a8.png" data-download-href="/uploads/short-url/85jQTZaCBpepQXlrT64qqR6QBuM.png?dl=1" title="Screen Shot 2020-02-08 at 9.03.28 pm.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/259753/181096165-83b743c2-bfc4-47df-a68a-a8c3af1cc4ce.png" alt="Screen Shot 2020-02-08 at 9.03.28 pm.png" data-base62-sha1="85jQTZaCBpepQXlrT64qqR6QBuM" width="690" height="351" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/38ab598293ab0c530d8e82dbd748b2c00a89a0a8_2_690x351.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/38ab598293ab0c530d8e82dbd748b2c00a89a0a8_2_1035x526.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/38ab598293ab0c530d8e82dbd748b2c00a89a0a8_2_1380x702.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/38ab598293ab0c530d8e82dbd748b2c00a89a0a8_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2020-02-08 at 9.03.28 pm.png</span><span class="informations">1534×782 96.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p><a href="https://github.com/moroshko/basis/pull/9" rel="nofollow noopener">Here</a> is a Pull Request example with the stuck build step.</p>
<p>Workflow file is <a href="https://github.com/moroshko/basis/blob/2cd9439e32c17cc112a0160a5df8633aa9d21238/.github/workflows/ci.yml" rel="nofollow noopener">here</a>.</p>
<p>How could I remove the 12.x build from the checks?</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/moroshko">@moroshko</a> ,</p>
<p>I can reproduce the same when I add ’12.x’ checkrun as required in master branch policy.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/e/e68fc7fc484f4a0e673d061bb44ddeee97c5e8e3.png" data-download-href="/uploads/short-url/wTDROy7waqj3H09PB95VKCTOQJd.png?dl=1" title="reproPR.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/52772761/181096167-c479f234-6a40-4378-84fe-268d60b1d24a.png" alt="reproPR.png" data-base62-sha1="wTDROy7waqj3H09PB95VKCTOQJd" width="690" height="246" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/e/e68fc7fc484f4a0e673d061bb44ddeee97c5e8e3_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">reproPR.png</span><span class="informations">742×265 17 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>After I remove the checkrun ‘build(12.x)’ in master branch policy, it fixes the error, PR passed now.</p>
<p><a href="https://github.com/%7Borg%7D/%7Brepo%7D/settings/branches" rel="nofollow noopener">https://github.com/{org}/{repo}/settings/branches</a></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/1/14c5d129debf289f3d2f812f6e50376d826256b2.png" data-download-href="/uploads/short-url/2XLmY6dAq357hwHAcaQJVFvZJT4.png?dl=1" title="Uncheckoption.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/52772761/181096168-2f6d0780-7023-404f-b4f5-d23b3b6e0d0f.png" alt="Uncheckoption.png" data-base62-sha1="2XLmY6dAq357hwHAcaQJVFvZJT4" width="690" height="282" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/1/14c5d129debf289f3d2f812f6e50376d826256b2_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Uncheckoption.png</span><span class="informations">738×302 18.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/b/b6ba4d838d0be53788dfb13168bc1962a9dd8636.png" data-download-href="/uploads/short-url/q4u3bZahNEDR8aTfYAXwVdnWEOW.png?dl=1" title="PRFixed.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/52772761/181096169-25e7086d-3397-4c63-b3ba-0da42d3ff1b2.png" alt="PRFixed.png" data-base62-sha1="q4u3bZahNEDR8aTfYAXwVdnWEOW" width="690" height="190" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/b/b6ba4d838d0be53788dfb13168bc1962a9dd8636_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">PRFixed.png</span><span class="informations">738×204 11.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Or you can delete the branch policy, create a new one and try again. Hope it helps!</p>
